TASK: Adds neos/neos-setup as dependency

markusguenther opened the pull request:
As we removed neos/neos-setup from neos/neos-development-collection dependencies, we need to declare it here. This change also updates the welcome command to setup, as welcome is just an alias for the previous CLI setup.

This is related to https://github.com/neos/neos-development-collection/pull/4803

mhsdesign commented:
actually when the package `neos/setup` is installed, this welcome script should still function.

Thought we might need a helping hand to only run this command if available ;)

markusguenther commented:
> actually when the package `neos/setup` is installed, this welcome script should still function.
> 
> Thought we might need a helping hand to only run this command if available ;)

Yes, but it is only an alias and as the other package is removed, it is maybe better to just use the new command.

markusguenther committed:
TASK: Adds neos/neos-setup as dependency

As we removed neos/neos-setup from neos/neos-development-collection dependencies, we need to declare it here. This change also updates the welcome command to setup, as welcome is just an alias for the previous CLI setup.

mhsdesign commented on composer.json:
```diff
@@ -51,7 +52,7 @@
   },
   "scripts": {
-    "post-create-project-cmd": "./flow welcome",
+    "post-create-project-cmd": "./flow setup:index",
```
```suggestion
    "post-create-project-cmd": "./flow setup",
```

ahaeslich replied:
So I guess to not confuse anyone the base distribution should also use `"post-create-project-cmd": "./flow setup",` instead of `"post-create-project-cmd": "./flow setup:index",`?

see: https://github.com/neos/neos-base-distribution/pull/61

mhsdesign replied:
Yes

mhsdesign replied:
And also target 8.0

markusguenther committed:
TASK: Simplify command for post-create-project-cmd

Co-authored-by: Marc Henry Schultz <85400359+mhsdesign@users.noreply.github.com>
